tracer/src/Datadog.Trace/RemoteConfigurationManagement/RcmCapabilitiesIndices.cs

```diff
@@ -9,39 +9,42 @@ namespace Datadog.Trace.RemoteConfigurationManagement
 {
     internal static class RcmCapabilitiesIndices
     {
-#pragma warning disable SA1203 // Constants should appear before fields
-        public const uint ReservedUInt32 = 0;
-        public static readonly BigInteger Reserved = new(ReservedUInt32);
+        public static readonly BigInteger Reserved = Create(0);
```

Comment:
Just want to double check but I think this would be creating `1` instead of `0`?
I think `1UL << 0` is `1`?

Reply:
That's interesting. You are absolutely right, but it means the *previous* logic was broken. `Reserved` is supposed to be bit 0, so the value should be 1.